lib/fetch/rss/fetchRssFeed.ts

```diff
@@ -16,14 +30,51 @@ class RssProxyError extends Error {
 /**
  * Fetch and parse RSS feed URL.
  * @param feedUrl URL to request feed from.
- * @returns Parsed RSS data object.
+ * @returns Promise for parsed IRSS data object.
  */
-const fetchRssFeed = async (feedUrl: string) => {
+const fetchRssFeed = async (feedUrl: string): Promise<IRss> => {
   try {
-    return await parser.parseURL(feedUrl);
+    const feed = await parser.parseURL(feedUrl);
+    return decoratePodcast(feed);
```

Comment:
Follows the same sort of pattern for how the `itunes` namespace is handled in the `rss-parser`:
https://github.com/rbren/rss-parser/blob/master/lib/parser.js#L179